.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
From the [Markdown documentation](https://daringfireball.net/projects/markdown/syntax):

> When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return.